### PR TITLE
tests: add a note explaining the UBUNTU_STORE_AUTH_DATA_FILENAME env var

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This is needed to understand how to regenerate the env var value and when to do it
This env var is used in remodel nested tests

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
I'll talk to the store to see if the

File: tests/nested/manual/remodel-cross-store/task.yaml [hunk 1]
```diff
@@ -41,6 +41,10 @@ prepare: |
     REMODEL_SNAP=$(ls test-snapd-remodel-pc_*.snap)
     mv "$REMODEL_SNAP" "$(tests.nested get extra-snaps-path)"
 
+    # Use UBUNTU_STORE_AUTH_DATA_FILENAME en var to authenticate against the store
```
Comment (MiguelPires) [inline tests/nested/manual/remodel-cross-store/task.yaml:44]:
```suggestion
    # Use UBUNTU_STORE_AUTH_DATA_FILENAME env var to authenticate against the store
```

File: tests/nested/manual/remodel-simple/task.yaml [hunk 2]
```diff
@@ -34,6 +34,10 @@ prepare: |
     REMODEL_SNAP=$(ls test-snapd-remodel-pc_*.snap)
     mv "$REMODEL_SNAP" "$(tests.nested get extra-snaps-path)"
 
+    # Use UBUNTU_STORE_AUTH_DATA_FILENAME en var to authenticate against the store
```
Comment (MiguelPires) [inline tests/nested/manual/remodel-simple/task.yaml:37]:
```suggestion
    # Use UBUNTU_STORE_AUTH_DATA_FILENAME env var to authenticate against the store
```

**Review by MiguelPires — APPROVED** [review]
Thanks

**Review by mvo5 — APPROVED** [review]
Thank you! This is really nice (but please check the suggestion from Miguel)

**Pull request merged**